test/pod_performance/README.md

```diff
@@ -0,0 +1,37 @@
+# Kueue Pod Integration Performance Testing
+## Introduction
+A minimal setup for performance testing Plain Pods integration using
+clusterloader2.
+
+## Setup
+
+Install Kueue, configured to use [Plain Pods](https://kueue.sigs.k8s.io/docs/tasks/run_plain_pods/).
+
+e.g.
+```
+wget https://github.com/kubernetes-sigs/kueue/releases/download/v0.6.2/manifests.yaml
+
+# manifest.diff is provided in this directory
+patch manifests.yaml manifest.diff
+kubectl apply --server-side -f manifests.yaml
+```
+
+Install and build clusterloader - see https://github.com/kubernetes-sigs/kueue/tree/main/test/performance#how-to-run-the-test
+Then, set the env variable `CL2_HOME_DIR` to its location.
+
+PROVIDER and KUBECONFIG env variables may also be overridden.
+
+## Running
+Parameters are configurable in test-config.yaml. Apart from clusterqueues,
+resources are created in the kueue-pod-performance-1 namespace.
+
+```
+CL2_HOME_DIR="/path/to/your/clusterloader" ./run-test.sh
+```
+
+## Cleanup
+```
+kubectl delete namespace/kueue-pod-performance-1
```

Comment:
why not do this in run-test.sh?

Reply:
This namespace is automatically deleted by clusterloader2. Additionally, we cleanup the clusterqueues and resourceflavor at the end of run-test.sh. However, if the user SIGINTs the test, this cleanup will not occur.

I added a note indicating as such